packages/common/normalizers/schemas/list.js

```diff
@@ -14,7 +18,12 @@ const listSchema = new schema.Entity(
       ...entity,
       __type: ENTITY_TYPE.LIST,
       teamId: entity.teamId || TEAM_TYPE.PERSONAL,
-      _permissions: createPermissionsFromLinks(entity._links),
+      _permissions: {
+        ...createPermissionsFromLinks(entity._links),
+        __typename: entity.teamId
```

Comment:
Might be broken.
please, rewrite it with `entity.teamId !== TEAM_TYPE.PERSONAL`.